src/saveload/vehicle_sl.cpp

```diff
@@ -699,7 +698,6 @@ class SlVehicleCommon : public DefaultSaveLoadHandler<SlVehicleCommon, Vehicle>
 		SLE_CONDVAR(Vehicle, profit_last_year,      SLE_INT64,                   SLV_65, SL_MAX_VERSION),
 		SLEG_CONDVAR("cargo_feeder_share", _cargo_feeder_share, SLE_FILE_I32 | SLE_VAR_I64,  SLV_51,  SLV_65),
 		SLEG_CONDVAR("cargo_feeder_share", _cargo_feeder_share, SLE_INT64,                   SLV_65,  SLV_68),
-		SLEG_CONDVAR("cargo_loaded_at_xy", _cargo_loaded_at_xy, SLE_UINT32,                  SLV_51,  SLV_68),
```

Comment:
Is this right for loading a [SLV_15..SLV_68) savegame?

Reply:
See `compat/vehicle_sl_compat.h` a bit higher:

`SLC_NULL(4, SLV_51, SLV_68),`

I think that should do it.